app/infrastructure/queue_consumer.py

```
@@ -162,7 +162,30 @@ async def _dequeue_job(
             return None
-
+    
+    # ===============================================================
+    # ================_process_job===================================
+    # ===============================================================
+    
+    async def _set_processing_tracker(self, tracker):
+        if tracker:
+            try:
+                await tracker.update_step(JobLevels.PROCESSING)
+            except Exception:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)